src/app/about/about.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit } from '@angular/core';
+import { DomSanitizer, SafeResourceUrl } from '@angular/platform-browser';
+import { OrganizationsService } from 'app/shared/swagger';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { Funder, FundingComponent } from '../funding/funding.component';
+import { Dockstore } from '../shared/dockstore.model';
+import { Sponsor } from '../sponsors/sponsor.model';
+
+@Component({
+  selector: 'app-about',
+  templateUrl: './about.component.html',
+  styleUrls: ['./about.component.css'],
+  providers: [FundingComponent],
+})
+export class AboutComponent implements OnInit {
+  Dockstore = Dockstore;
+  private readonly BROAD_ORGANIZATION_ID = 16;
+  private readonly BROAD_COLLECTION_ID = 23;
+  public youtubeSafeURL: SafeResourceUrl;
+  public workflowsLength$: Observable<number>;
+  public funders: Funder[];
+  public partners: Sponsor[] = [
+    new Sponsor('dnastack.png', new URL('https://dnastack.com')),
+    new Sponsor('sevenbridges.png', new URL('https://www.sevenbridges.com')),
+    new Sponsor('terra.png', new URL('https://terra.bio')),
+    new Sponsor('dnanexus.png', new URL('https://www.dnanexus.com/')),
+    new Sponsor('anvil.png', new URL('https://anvilproject.org/')),
```

Comment:
Does this end up on the funding page (but funders is empty on line 22)?

I feel like this list isn't correct. Why is anvilproject, a grant, mixed in with the platforms? And where are BD Catalyst and BISTI (didn't check Canadian grants)?

Your screenshots look good, but I'm confused by the code.

Reply:
`funders` is initialized in `ngOnInit()` at line 50
Both BD Catalyst and BISTI are inside `funders`, and anvil was in the partners section in the mock. If it's needed I could move them around into different columns, just let me know where to
https://app.zeplin.io/project/5fc690a2717ee9144814f190/screen/60074bdcbb1d4b5c92a996d3

Reply:
This does in fact match the mock @coverbeck I think integration partners was meant to be basically be the launch with partners. 

That said, we can tweak but I think I prefer to get this merged and then tweak in subsequent PRs, with the initial PRs just having the target of matching the mock-ups.  That said, @wendidy can you replace the BD Catalyst logo with a plain text image like literally something from https://www.coolgenerator.com/text-image-generator 

After the mock-up was created we found out we don't have permission to use the BD Catalyst logo.

Reply:
@denis-yuen Do I need to switch out the BD Catalyst image on `/funding` as well?

Reply:
Yes, I believe so and thanks